src/VisualStudio/Core/Def/ColorSchemes/VisualStudio2019.xml

```diff
@@ -564,6 +600,18 @@
                 <Background Type="CT_RAW" Source="FF3E3E3E" />
                 <Foreground Type="CT_RAW" Source="FFA9A8A7" />
             </Color>
+            <Color Name="inline diagnostics - syntax error">
```

Comment:
Worth adding a comment here and on syntax errors that if one default changes the other should change?

Reply:
+ others as well

Reply:
I do not have the code currently doing that because I thought full customizability should be allowed. Should changing the colors of the diagnostic also change the color of the squiggle?

Reply:
It doesn't need to always be the same, but the defaults should be the same right? It will be hard for someone to know if this needs to be updated with changes to other colors.

Reply:
Oh I see, I will add comments about that.